src/coreclr/pal/src/arch/riscv64/context2.S

```diff
@@ -9,4 +9,238 @@
 #include "unixasmmacros.inc"
 #include "asmconstants.h"
 
-#error "TODO-RISCV64: missing implementation"
+// Incoming:
+//  a0: Context*
+//  a1: Exception*
+//
+LEAF_ENTRY RtlRestoreContext, _TEXT
+#ifdef HAS_ASAN
+// TODO-RISCV64: missing implementation
```

Comment:
It's better to add build error similar to the one in exceptionhelper.S:
```
#pragma error("LLVM v3.9 ASAN unimplemented on RISCV64 yet")
```

Reply:
Fixed